x-pack/legacy/plugins/infra/public/pages/logs/analysis/page_results_content.tsx

```diff
@@ -60,7 +65,12 @@ export const AnalysisResultsContent = ({
   const [queryTimeRange, setQueryTimeRange] = useState<TimeRange>(
     stringToNumericTimeRange(selectedTimeRange)
   );
-
+  const getMlLink = getMlLinkFormatter({
+    basePath,
+    jobId,
+    startTime: queryTimeRange.startTime,
+    endTime: queryTimeRange.endTime,
+  });
```

Comment:
Could we keep this function reference as constant as possible using `useMemo` or `useCallback`?